frontend/src/pages/auth/components/not-found/not-found.tsx

```diff
@@ -0,0 +1,26 @@
+import { useRouteError } from 'react-router-dom';
```

Comment:
Make export for this hook from `~libs/hooks/hooks` (that's the place for exporting common hooks)